modules/katamari/src/test/ts/atomic/api/str/CapitalizeTest.ts

```diff
@@ -1,36 +1,36 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import { Testable } from '@ephox/dispute';
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
 import fc from 'fast-check';
 import * as Strings from 'ephox/katamari/api/Strings';
 
-const { tString } = Testable;
+describe('atomic.katamari.api.str.CapitalizeTest', () => {
+  it('unit tests', () => {
+    const check = (expected, input) => {
```

Comment:
```suggestion
    const check = (expected: string, input: string) => {
```
Not related to this change, so feel free to ignore.